cognite_toolkit/_cdf_tk/utils/hashing.py

```diff
@@ -42,15 +42,11 @@ def calculate_secure_hash(item: dict[str, Any]) -> str:
 
 
 def calculate_str_or_file_hash(content: str | Path, shorten: bool = False) -> str:
-    sha256_hash = hashlib.sha256()
-    if isinstance(content, Path):
-        content = content.read_text("utf-8")
-    # Get rid of Windows line endings to make the hash consistent across platforms.
-    sha256_hash.update(content.encode("utf-8").replace(b"\r\n", b"\n"))
-    calculated = sha256_hash.hexdigest()
-    if shorten:
-        return calculated[:8]
-    return calculated
+    if isinstance(content, str):
+        byte_content = content.encode("utf-8")
+    else:
+        byte_content = content.read_bytes()
```

Comment:
It is much safer to just hash the bytes directly and not care about the encoding of the file